schema/field/field.go

```diff
@@ -895,6 +916,131 @@ func (b *uuidBuilder) Descriptor() *Descriptor {
+}
+
+// Unique makes the field unique within all vertices of this type.
+func (b *otherBuilder) Unique() *otherBuilder {
+	b.desc.Unique = true
+	return b
+}
+
+// Sensitive fields not printable and not serializable.
+func (b *otherBuilder) Sensitive() *otherBuilder {
+	b.desc.Sensitive = true
+	return b
+}
+
+// Default sets the default value of the field. For example:
+//
+//	field.Other("link", &Link{}).
+//		SchemaType(map[string]string{
+//			dialect.MySQL:    "text",
+//			dialect.Postgres: "varchar",
+//		}).
+//		// A static default value.
+//		Default(&Link{Addr: "0.0.0.0"})
+//
+//	field.Other("link", &Link{}).
+//		SchemaType(map[string]string{
+//			dialect.MySQL:    "text",
+//			dialect.Postgres: "varchar",
+//		}).
+//		// A function for generating the default value.
+//		Default(NewLink)
+//
+func (b *otherBuilder) Default(v interface{}) *otherBuilder {
+	b.desc.Default = v
+	switch fieldT, defaultT := b.desc.Info.RType.rtype, reflect.TypeOf(v); {
+	case fieldT == defaultT:
+	case defaultT.Kind() == reflect.Func:
+		b.desc.checkDefaultFunc(b.desc.Info.RType.rtype)
+	default:
+		b.desc.Err = fmt.Errorf("expect type (func() %[1]s) or (%[1]s) for other default value", b.desc.Info)
+	}
+	return b
+}
+
+// Nillable indicates that this field is a nillable.
+// Unlike "Optional" only fields, "Nillable" fields are pointers in the generated field.
+func (b *otherBuilder) Nillable() *otherBuilder {
+	b.desc.Nillable = true
+	return b
+}
+
+// Optional indicates that this field is optional on create.
+// Unlike edges, fields are required by default.
+func (b *otherBuilder) Optional() *otherBuilder {
+	b.desc.Optional = true
+	return b
+}
+
+// Immutable indicates that this field cannot be updated.
+func (b *otherBuilder) Immutable() *otherBuilder {
+	b.desc.Immutable = true
+	return b
+}
+
+// Comment sets the comment of the field.
+func (b *otherBuilder) Comment(c string) *otherBuilder {
+	b.desc.Comment = c
+	return b
+}
+
+// StructTag sets the struct tag of the field.
+func (b *otherBuilder) StructTag(s string) *otherBuilder {
+	b.desc.Tag = s
+	return b
+}
+
+// StorageKey sets the storage key of the field.
+// In SQL dialects is the column name and Gremlin is the property.
+func (b *otherBuilder) StorageKey(key string) *otherBuilder {
+	b.desc.StorageKey = key
+	return b
+}
+
+// SchemaType overrides the default database type with a custom
+// schema type (per dialect) for string.
+//
+//	field.Other("link", Link{}).
+//		SchemaType(map[string]string{
+//			dialect.MySQL:    "text",
+//			dialect.Postgres: "varchar",
+//		})
+//
+func (b *otherBuilder) SchemaType(types map[string]string) *otherBuilder {
+	b.desc.SchemaType = types
+	return b
+}
+
+// Annotations adds a list of annotations to the field object to be used by
+// codegen extensions.
+//
+//	field.Other("link", &Link{}).
+//		SchemaType(map[string]string{
+//			dialect.MySQL:    "text",
+//			dialect.Postgres: "varchar",
+//		}).
+//		Annotations(entgql.Config{
```

Comment:
I pulled down the latest ent-contrib and I'm not seeing the same annotation api as this doc comment. Instead I see something like:
```
Annotations(
				entgql.OrderField("LINK_FIELD"),
			).
```

Reply:
You're right. This was change added by https://github.com/facebook/ent/pull/1145 and I missed this.

Thanks for adding your review, it's really appreciated 🙏